docs/lang/char.md

```diff
@@ -0,0 +1,35 @@
+Character
+==============
+
+Types
+-----
+
+The character extension adds one new base type:
+
+    "char"
+
+Characters are [UTF-16][] code units.
```

Comment:
This is totally fine and no need to change it for now, but someday we might want to instead specify that this is any single Unicode character instead. Tying stuff to UTF-16 is a little annoying, even though it's what we'd inherit from JavaScript.

Reply:
Ok